RentACar/RentACar/Repositories/ClientRepository.cs

```diff
@@ -42,5 +63,10 @@ protected override void UpdateEntity(Client existingEntity, Client newEntity)
             existingEntity.Province = newEntity.Province;
             existingEntity.LastUpdate = DateTime.UtcNow;
         }
+
+        private bool DniExistInList(string dni)
+        {
+            return EntityList.Any(e => e.Dni == dni);
+        }
```

Comment:
Does this method make sense? I know it's just one line, but I think it's more readable

Reply:
it's OK if you want to leave it, but right now I, personal opinion, would drop it